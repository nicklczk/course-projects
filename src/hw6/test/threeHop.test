# Tests path finding for three hops

# Create graph
LoadGraph friends champions.tsv

# find path three hops
FindPath friends Kevin Kyle