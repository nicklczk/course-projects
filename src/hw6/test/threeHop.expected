# Tests path finding for three hops

# Create graph
loaded graph friends

# find path three hops
path from Kevin to Kyle:
Kevin to Eya via Apartments
Eya to Candice via Lander
Candice to Kyle via McMahon